plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/Plugin.java

```diff
@@ -0,0 +1,77 @@
+package io.jenkins.tools.pluginmanager.impl;
+
+import java.util.jar.JarFile;
+
+public class Plugin {
+    private String name;
+    private String version;
+    private String url;
+    private String archiveFileName;
+    private JarFile jarFile;
+    private boolean isPluginOptional;
+
+
+    public Plugin(String name, String version, String url) {
+        this.name = name;
+        this.version = version;
+        this.url = url;
+        this.archiveFileName = new StringBuffer(name).append(".jpi").toString();
+
+    }
+
+    public Plugin(String name, String version, boolean isPluginOptional) {
+        this.name = name;
+        this.version = version;
+        this.isPluginOptional = isPluginOptional;
+        this.archiveFileName = new StringBuffer(name).append(".jpi").toString();
```

Comment:
same as above

Reply:
Resolve in 11d31640d469a94ef77675cc77c18eeeb8b133b1